builtin/logical/pki/path_fetch.go

```diff
@@ -126,7 +126,23 @@ hyphen-separated octal`,
 // This returns the CRL in a non-raw format
 func pathFetchCRLViaCertPath(b *backend) *framework.Path {
 	return &framework.Path{
-		Pattern: `cert/(crl|delta-crl|unified-crl|unified-delta-crl)`,
+		Pattern: `cert/(crl|delta-crl)`,
```

Comment:
Does this work? I thought for some reason these only worked if the regexes didn't overlap. 

Or maybe it just adds it to the OpenAPI data, but they all use the same handler... Hmmm...